agent/app/agent_unspecified.go

```diff
@@ -13,15 +13,15 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package ecsclient
+package app
 
-import "os"
+import (
+	"errors"
 
-// initPID defines the process identifier for the init process
-const initPID = 1
+	"github.com/aws/amazon-ecs-agent/agent/engine"
+	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+)
 
-// isAgentAlsoTheInit returns true if the Agent process's pid is
-// 1, which means it's running without an init system
-func isAgentAlsoTheInit() (bool, error) {
-	return os.Getpid() == initPID, nil
+func (agent *ecsAgent) initializeTaskENIDependencies(state dockerstate.TaskEngineState, taskEngine engine.TaskEngine) (error, bool) {
+	return errors.New("unsupported platform"), true
```

Comment:
out of curiosity - how are we deciding to use one over the other? are we using `github.com/pkg/errors` only when we want to include additional context?